src/rustdoc_schema.graphql

```diff
@@ -27,6 +27,8 @@ type Crate {
   includes_private: Boolean!
   format_version: Int!
 
+  root_module: Module!
```

Comment:
None of the other things have docs, which was just me being sloppy early on. But it's good to start going forward, and here it's easy enough.
```suggestion
  """
  The top-level module of the crate, in which everything else in the crate is contained.
  """
  root_module: Module!
```